GDJS/Runtime/CustomRuntimeObject.ts

```diff
@@ -56,19 +59,24 @@ namespace gdjs {
         parent,
         this
       );
+      this._renderer = this._createRender();
 
       this._instanceContainer.loadFrom(objectData);
-      this.getRenderer().reinitialize(this, parent);
 
       // The generated code calls onCreated at the constructor end
       // and onCreated calls its super implementation at its end.
     }
 
+    protected abstract _createRender():
+      | gdjs.CustomRuntimeObject2DRenderer
+      | gdjs.CustomRuntimeObject3DRenderer;
+    protected abstract _reinitializeRender(): void;
```

Comment:
`_reinitializeRender` => `_reinitializeRenderer`

to be very precise on what this does.